TypeCobol.Test/SemanticDomainTest.cs

```diff
@@ -0,0 +1,1720 @@
+﻿using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Test.Compiler.Parser;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.IO;
+using TypeCobol.Codegen.Config;
+using TypeCobol.Compiler.Diagnostics;
+using TypeCobol.Compiler.Directives;
+using TypeCobol.Test.Utils;
+using System.Text;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Tools.Options_Config;
+using TypeCobol.Compiler.Domain;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.CodeModel;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Compiler.Symbols;
+using TypeCobol.Compiler.Types;
+using Type = TypeCobol.Compiler.Types.Type;
+
+namespace TypeCobol.Test.Domain
+{
+    [TestClass]
+    public class SemanticDomainTest
+    {
+        public static TypeCobolConfiguration DefaultConfig = null;
+        public static ProgramSymbolTableBuilder Builder = null;
+        public static NodeListenerFactory BuilderNodeListenerFactory = null;
+        public static string DefaultIntrinsicPath = null;
+
+        [TestInitialize]
+        public void TestInitialize()
```

Comment:
This is not a real shared method as you can see it have a test attribute [TestInitialize] It belongs to the set of tests inside this class, and it use static fields for this tests, if we use this in another test file it will conflict, and produced bad tests results, the same observation can be made for
  [TestCleanup]
        public void TestCleanup()